### ISSUE TITLE
glpi_monitoring 0.84+1.3i et glpi 0.84.6 graph bug

### ISSUE THREAD

**Issue by hahahafr** [ddurieux/glpi_monitoring#213]
Avec glpi_monitoring 0.84+1.3i et glpi 0.84.6 j'ai ce bug avec les graphiques 12h et 24h : 
![bug_glpi_monitoring_graphes](https://cloud.githubusercontent.com/assets/330148/13426653/19a7a38e-dfaf-11e5-8fa6-8efdfdc77827.jpg)
On dirai que le graphe boucle et est décalé vers la gauche.
Etrange.
Firefox 44.0.2

**Comment by ddurieux** [comment]
Yep, ça me le fais des fois, mai sje n'ai pas réussi à corriger, peut être en mettant à jour la lib qui fait les graphs
